src/foamadapter/solver/icofoam.py

```diff
@@ -0,0 +1,101 @@
+import pybFoam
+from pybFoam import (
+    Info,
+    Time,
+    Word,
+    adjustPhi,
+    constrainHbyA,
+    constrainPressure,
+    createPhi,
+    dictionary,
+    fvc,
+    fvm,
+    fvMesh,
+    fvScalarMatrix,
+    fvVectorMatrix,
+    pisoControl,
+    setRefCell,
+    solve,
+    surfaceScalarField,
+    volScalarField,
+    volVectorField,
+)
```

Comment:
These long list of imports are horrible. We could

```suggestion
import pybFoam as pyf
)
```

and then use `pyf.Info`  etc, because we a way to add neon bindings without conflicts as well. Or just do a `from pybFoam import *`

Reply:
simplified

Reply:
simplified the imports